platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/chatter/protocol/messages/ChatterEventDescriptor.java

```diff
@@ -157,6 +157,6 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return String.format("GossipEventDescriptor(%d,%d,%s)", creator, generation, CommonUtils.hex(hash.getValue()));
+        return "GossipEventDescriptor(%d,%d,%s)".formatted(creator, generation, CommonUtils.hex(hash.getValue()));
```

Comment:
Why is this one not changed to use ToStringBuilder?

Reply:
Mainly because there is no built-in format in `ToStringBuilder` that would preserve this formatting.

Reply:
It is safe for you to modify this format, this `toString()` is for debugging purposes.